friends.html

```diff
@@ -0,0 +1,161 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <title>Developers Rising</title>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/materialize/0.100.1/css/materialize.min.css">
+  <link rel="stylesheet" href="./styles.css">
+  <link href="https://fonts.googleapis.com/css?family=Varela+Round" rel="stylesheet">
+</head>
+
+<body>
+
+  <div class="navbar-fixed">
+    <nav>
+      <div class="nav-wrapper grey">
+        <a href="./index.html" class="brand-logo">Developers Rising</a>
+        <ul id="nav-mobile" class="right hide-on-med-and-down">
+          <li><a href="https://www.meetup.com/DevelopersRising/">Events</a></li>
+          <li><a href="./friends.html">Team</a></li>
+          <li><a href="./members.html">Members</a></li>
+          <li><a href="./legal.html">Legal</a></li>
+          
+        </ul>
+      </div>
+    </nav>
+  </div>
+
+  <div class="section grey">
+    <div class="row container center">
+      <h3 class="grey-text text-darken-4">Team</h3>
+      <h5>Meet our cotributers</h5>
+      <br />
+      <br />
+      <div class="row valign-wrapper">
+        <div class="row">
+          <div class="col s2 offset-s2">
+            <a href="http://www.createdd.com/">
+            <img src="https://avatars3.githubusercontent.com/u/22077628?v=4&s=400" alt="Avatar of Daniel Deutsch" class="circle responsive-img">
+          </a>
+          </div>
+          <div class="col s6">
+            <h4>Daniel Deutsch <b>(organizer)</b></h4>  
+              Aspiring Web Developer with Business Law Background. Pushing the limits to make the world a better place.
+          </div>
+        </div>
+
+      </div>
+      <div class="row valign-wrapper">
+          <div class="row">
+            <div class="col s2 offset-s2">
+              <a href="http://www.createdd.com/">
+              <img src="https://avatars3.githubusercontent.com/u/13132899?v=3&s=460" alt="Avatar of Daniel Deutsch" class="circle responsive-img">
+            </a>
+            </div>
+            <div class="col s6">
+              <h4>Robert Axelsen<b> (founder and organizer)</b></h4>  
+              Software Developer and Public Speaker, passionate about Open Source, Community and Education. Life-long learner.
+            </div>
+          </div>
+  
+        </div>
+    </div>
+  </div>
+
+  <div class="parallax-container">
+    <div class="parallax"><img src="img/1.jpg"
+        alt="Buisness handshake" /></div>
+  </div>
+
+  <div class="section grey">
+    <div class="row container center">
+      <h3 class="grey-text text-darken-4">Partners</h3>
+      <h5>Meet our partners</h5>
+    </div>
+  </div>
+
+  </div>
+  </div>
+  <div class="parallax-container">
```

Comment:
Could you please provide the source link. With the relevant license and author 👍